config/templates/cloud-config-worker

```diff
@@ -29,8 +29,8 @@ coreos:
           {{.AWSCliImageRepo}}:{{.AWSCliTag}} -- cfn-init -v \
               -c "etcd-client" \
               --region {{.Region}} \
-              --resource LaunchConfigurationController \
-              --stack {{ .ClusterName }}
```

Comment:
Sounds great!
However `cfn-init` doesn't seem to have access to a stack other than the one in which the EC2 instance running `cfn-init` is directly/indrectly managed 😢 
`cfn-init` fails like:

```
Jan 18 23:46:32 ip-10-0-1-242.ap-northeast-1.compute.internal rkt[7989]: [ 1044.115954] awscli[5]: ValidationError: Instance i-0507f6146f3c1bd79 in not a member of stack arn:aws:cloudformation:ap-northeast-1:<account-id>:stack/kubeawstest2/4dfa4280-dd8e-11e6-8dbb-500c44f24ce6
```